resources/compass/charts/kyma-environment-broker/values.yaml

```diff
@@ -36,14 +37,14 @@ provisioner:
   URL: "http://compass-provisioner.compass-system.svc.cluster.local:3000/graphql"
   gardener:
     # name of the secret with kubeconfig to the gardener cluster
-    secretName: "gardener"
+    secretName: "gopher-canary-gardener"
```

Comment:
should we really have `gopher` in values? 😛  @polskikiel @mszostok @piotrmiskiewicz @jasiu001